Automattic/wp-calypso
# Add Jetpack Cloud Universal Text Styles and use in Activity Log

#### Changes proposed in this Pull Request

* Add universal `h2`, `h3`, `p`, and `a` styles to Jetpack Cloud
* use to apply text styling to `/backups/activity`

![41101 before:after](https://user-images.githubusercontent.com/2810519/79594454-e3484900-8091-11ea-8fcb-48412d190528.jpg)


#### Testing instructions

1. Navigate to `/backups/activity`
2. Check that the style of the text matched design
3. No functional changes
